ctapipe/io/simteleventsource.py

```diff
@@ -430,23 +432,25 @@ def _generate_events(self):
             yield data
 
     @staticmethod
-    def _fill_event_pointing(pointing, tracking_position):
+    def _fill_event_pointing(tracking_position):
```

Comment:
this is not really a "fill" method anymore, is it?

Reply:
depends. Fills the container...